### PR TITLE
Brief introdcution of ATSSS

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -74,49 +77,48 @@ as shown here.
...
 
+# Terminology 
+
+This document makes use of 3GPP-specific terms defined in RFC6459, mainly the following ones: 
+
+-    Packet Data Network (PDN): is a packet-based network that
+      either belongs to the operator or is an external network such as
+      the Internet or a corporate intranet.  The user eventually
+      accesses services in one or more PDNs.  The operator's packet core
+      networks are separated from packet data networks either by GGSNs, PDN Gateways (PDN-GWs), or User Plane Function (UPF).
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd]: Do we need to introduce GGSN and PDN-GW, while the ATSSS is restricted to the 5GC?

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: OK to remove them. 

Fixed in: https://github.com/obonaventure/draft-quic-atsss-reqs/pull/11/commits/6f57252ed60a2b5106590ad4cd4dad8e4dff82e8

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -74,49 +77,48 @@ as shown here.
...
-  +------ Net 1 ------+
-  |                   |
-+-+-+               +-+-+                     +---+
-| A |               | M |----- Internet  -----+ B |
-+-+-+               +-+-+                     +---+
-  |                   |
-  +------ Net 2 ------+
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-{: #fig-env title="Reference environment"}
+An UE can be provided a multi-access PDU Connectivity Service. That is, an UA can exchange data with a PDN by using a "3GPP access network" and a "non-3GPP access network". Such service relies upon the The ATSSS feature.
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd]: "UE" instead of "UA"

File: draft-quic-atsss-reqs.mkd [hunk 3]
```diff
@@ -74,49 +77,48 @@ as shown here.
...
-is connected to the Internet. B does not use multipath transport protocols
-while A would like to be able to use its two access networks to reach B. To
-enable A to use its two paths efficiently, a possible solution is to use
-a midpoint node, called M in the figure, so that multipath transport is
-used between A and M while regular (single path) transport is used
-between A and B. 
+The 5G Core (5GC) supports a service that provides exchange of data between a UE and a data network identified by a Data Network Name (DNN). Such connectivity service, called PDU Connectivity Service, is supported via 'PDU Sessions' that are established upon request from the UE. The type of PDU Session can be IPv4, IPv6, IPv4v6, Ethernet or Unstructured.
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is out of the scope of this document to provide a comprehensive overview of 5G System (5GS) architecture. In particuler, this document won't describe how PDU Sessions are established, and thus how IP addresses/prefixes are assigned to requetsing User Equipment (UEs).
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd]: "particular" instead of "particuler"

File: draft-quic-atsss-reqs.mkd [hunk 4]
```diff
@@ -74,49 +77,48 @@ as shown here.
...
+-    Packet Data Network (PDN): is a packet-based network that
+      either belongs to the operator or is an external network such as
+      the Internet or a corporate intranet.  The user eventually
+      accesses services in one or more PDNs.  The operator's packet core
+      networks are separated from packet data networks either by GGSNs, PDN Gateways (PDN-GWs), or User Plane Function (UPF).
+
+-     UE (User Equipment): refers to the devices that are hosts with the
+      ability to obtain Internet connectivity via a 3GPP network.  A MS
+      is comprised of the Terminal Equipment (TE) and a Mobile Terminal
+      (MT).
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd]: The terminologies MS, TE and MT seem GSM/2G specific.

File: draft-quic-atsss-reqs.mkd (hunk 2 above)
Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: fixed in 714b2d1

File: draft-quic-atsss-reqs.mkd (hunk 3 above)
Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: Fixed now in 54ecab3. Thanks.

File: draft-quic-atsss-reqs.mkd (hunk 4 above)
Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: That was to cover tethering and dongle cases, but this is fair. 

Fixed now in 54ecab3. Thank you.

**Pull request merged**